src/utilities/new-sequence-editor/command-dictionary.ts

```diff
@@ -37,7 +37,7 @@ export function fswCommandArgDefault(fswCommandArg: FswCommandArgument, enumMap:
       // If the enum symbol has whitespace, return it in quotes so it does not mess everything up.
       return `"${enumSymbolValue}"`;
     } else {
-      return enumSymbolValue ?? 'UNKNOWN_ENUM';
+      return `"${enumSymbolValue}"` ?? 'UNKNOWN_ENUM';
```

Comment:
This will always evaluate to return the first case because regardless of what the value of `enumSymbolValue` is, it will always be a string with at least `""`. The `??` also only checks for `null` or `undefined`. Instead, you might need to do:

```enumSymbolValue ? `"${enumSymbolValue}"` : 'UNKNOWN_ENUM'```